lib/miq_expression.rb

```diff
@@ -816,7 +809,9 @@ def self.quote_human(val, typ)
       else
         val
       end
-    when "string", "date", "datetime"
+    when nil
+      "\"string\""
+    when :string, :date, :datetime
```

Comment:
Similar comment to https://github.com/ManageIQ/manageiq/pull/22519/files#r1207226074